packages/react/src/unstable.ts

```diff
@@ -18,6 +18,9 @@ export * from "./command-input";
 export * from "./command-item";
 export * from "./command-list";
 export * from "./command-separator";
+export * from "./dropdown-sub";
+export * from "./dropdown-sub-content";
+export * from "./dropdown-sub-trigger";
```

Comment:
these should be called `DropdownMenu` + `Sub`
```suggestion
export * from "./dropdown-menu-sub";
export * from "./dropdown-menu-sub-content";
export * from "./dropdown-menu-sub-trigger";
```